src/main/java/com/google/cloud/tools/libraries/libraries.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0">
+<!-- Warning: non-normative. Probably not even correct. -->
+<apis>
+
+  <api name="BigQuery API v2"
+      definition="https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest"
```

Comment:
I'm guessing you're thinking this would be the source for API metadata:
```
{
 "kind": "discovery#restDescription",
 "etag": "\"YWOzh2SDasdU84ArJnpYek-OMdg/wbl_PWLcOQ9nI3adoCABAX0E-tY\"",
 "discoveryVersion": "v1",
 "id": "bigquery:v2",
 "name": "bigquery",
 "version": "v2",
 "revision": "20170721",
 "title": "BigQuery API",
 "description": "A data platform for customers to create, manage, share and query data.",
 "ownerDomain": "google.com",
 "ownerName": "Google",
 "icons": {
  "x16": "https://www.google.com/images/icons/product/search-16.gif",
  "x32": "https://www.google.com/images/icons/product/search-32.gif"
 },
 "documentationLink": "https://cloud.google.com/bigquery/",
```
This `definition` page is for REST — what about RPC?